website/content/docs/k8s/connect/api-gateway.mdx

```diff
@@ -0,0 +1,171 @@
+| `-log-level`                                                                                               | String value that specifies the logging level. The following values are supported: <br/>- `trace` (highest level of detail) <br/>- `debug` <br/>- `info` <br/>- `warn` <br/>- `error` | context to use when starting the Consul server. | Required                                                  | `info` |
+| `-metrics-port`                                                                                            | Integer value that specifies the port number for collecting metrics.                                                                                                                  | Optional                                        | none                                                      |
+| `-pprof`                                                                                                   | Integer value that specifies the Go pprof port number for collecting metrics.                                                                                                         | Optional                                        | none                                                      |
+| `-sds-server-host`                                                                                         | String value that specifies the host server for the secret discovery service (SDS).                                                                                                   | Optional                                        | `consul-api-gateway-controller.default.svc.cluster.local` |
+| `-sds-server-host`                                                                                         | Integer value that specifies the port number for the secret discovery service (SDS).                                                                                                  | Optional                                        | `9090`                                                    |
+
+You can also issue the `version` command to print the Consul API Gateway version to the console:
+
+```shell-session
+./consul-api-gateway version
+consul-api-gateway 0.1.0-dev
+```
+
+## Configuration
+
+Create the following artifacts to configure the API Gateway:
+
+- [Gateway](#gateway): Defines the gateway properties, including listeners.
+- [Listeners](#listeners): Defines listener properties, such as protocol, port, and namespace.
+- [Routes](#routes): Specifies the path from the client to the listener.
+
+### Gateway
+
+The gateway object contains gateway listeners.
+Add the `kind: Gateway` option to the configuration file to declare a gateway.
```

Comment:
@trujillo-adam you can highlight `kind: Gateway` if you use the new code block component and the highlight feature. 

https://react-components-hashicorp.vercel.app/components/codeblock

Example: # used to display example accurately

```
<CodeBlockConfig lineNumbers highlight="3,5">
 
#```javascript
console.log('Hello world!')
console.log('This is a second line.')
// Doing more stuff
function logAThirdLine() {
  console.log('Hello again')
}
#```
 
</CodeBlockConfig>
```